### DISCUSSION TITLE
Table Editable Attribute Issue

### DISCUSSION THREAD

**Discussion by james-chao** [dotnetcore/BootstrapBlazor#3249] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

升級8.4.1 後造成Table 欄位無法呈現，原因是Table Attribute 設定AutoGenerateColumns="false" Column Attribute 設定成Editable="false" 後造成，原先8.4.0 是正常呈現
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/3213919/bf8e6111-a800-4887-b029-0375ce684b53)

### Expected Behavior

將Table Attribute 設定AutoGenerateColumns="false" Column Attribute 設定成Editable="true"  後正常如下:
預期是Table可以呈現此欄位，但編輯Form 無法看到此欄位
<img width="948" alt="image" src="https://github.com/dotnetcore/BootstrapBlazor/assets/3213919/ac005d1d-7bb1-41a6-9629-eb2213aa299f">

### Steps To Reproduce

https://github.com/james-chao/BootstrapBlazorApp1.git

### .NET Version

8.0.202

### Anything else?

<Table TItem="Foo" IsPagination="true" PageItemsSource="PageItemsSource" class="table-demo"
       IsStriped="true" IsBordered="true" TableSize="TableSize.Compact" ShowToolbar="true" ShowDefaultButtons="false" ShowExtendDeleteButton="false" ShowRefresh="false"
       ShowLineNo="true" LineNoText="No." ShowEmpty="true" ShowSearch="true" ShowSearchText="false" SearchMode="SearchMode.Popup" AutoGenerateColumns="false" style="height:auto"
       ShowLoadingInFirstRender="true" HeaderStyle="TableHeaderStyle.Light" AllowResizing="true" ShowColumnList="false" IsMultipleSelect="false" ShowExtendButtons="true">
       <TableColumns>
        <TableColumn @bind-Field="@context.DateTime" FormatString="yyyy-MM-dd" Text="日期" Width="100"/>
        <TableColumn @bind-Field="@context.Count" Text="数量" Sortable="true" Searchable="true" Filterable="true" Editable="false" Align="Alignment.Left" Width="200" />
        <TableColumn @bind-Field="@context.Address" Text="地址" Sortable="true" Searchable="true" Filterable="true" Readonly="true" Align="Alignment.Left" Width="200" />
        <TableColumn @bind-Field="@context.Education" Text="学历" Width="200" />
        <TableColumn @bind-Field="@context.Hobby" Items="GetHobbys(context)" />
    </TableColumns>
</Table>

**Comment by bb-auto** [comment]
@james-chao Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment, marked as answer]
@james-chao Column Attribute 設定成 `Editable="false"` 现在就是不应该显示的，这没问题。老版本代码没有检查这个参数

现在的逻辑是 `TableColumn` 的 `Editable` 等同于 `AutoGenerateColumn` 中的 `Ignore` 设置后就是不显示，并且不会出现在 `ColumnList` 中

设置 `Visible="false"` 不显示但是会出现在 `ColumnList` 中，通过设置 `IsVisibleWhenAdd="true"` 新建时可编辑 设置 `IsVisibleWhenEdit` 更新时可编辑

**Reply by james-chao** [reply]
@ArgoZhang 感謝了，可以Work
